packages/voyager-keycloak/src/schemaDirectives/hasRole.ts

```diff
@@ -24,7 +24,7 @@ export class HasRoleDirective extends SchemaDirectiveVisitor {
   public visitFieldDefinition (field: any) {
     const { resolve = defaultFieldResolver } = field
     const { error, value } = this.validateArgs()
-
+    // TODO ignore for subscriptions
```

Comment:
?? :)

Reply:
i think dara has another pr that actually escapes this, need to check and I will remove this comment